bpfd/src/multiprog/tc.rs

```diff
@@ -28,9 +28,8 @@ use crate::{
     errors::BpfdError,
 };
 
-const DEFAULT_PRIORITY: u32 = 50;
-const MIN_TC_DISPATCHER_PRIORITY: u16 = 50;
-const MAX_TC_DISPATCHER_PRIORITY: u16 = 49;
+const DEFAULT_PRIORITY: u32 = 50; // Default priority for user programs in the dispatcher
+const TC_DISPATCHER_PRIORITY: u16 = 50; // Default TC priority for TC Dispatcher
```

Comment:
Is the `TC_DISPATCHER_PRIORITY` the priority of the actual dispatcher program itself? And if so shouldn't we be setting it at a lower one?

Reply:
Yes, `TC_DISPATCHER_PRIORITY` is the priority of the dispatcher.  The range is 1-64K, with lower number = higher priority, so 50 is pretty high, but I wanted to leave some higher priority slots as well.

Reply:
I'm alright with it for now just curious